### DISCUSSION TITLE
Can you share your configuration?

### DISCUSSION THREAD

**Discussion by ISouthRain** [nvim-neorg/neorg#325] [Q&A]
Hello !!
I am sad 😥
I have installed Plug neorg, plenary.nvim, and  nvim-treesitter ！！
My nvim-treesitter have install norg, norg_meta, norg_table !!
But Neorg don't work!!
I tried many times to install and configure it, but when I open the Norg file, I always have an error, and norg don't work.

**So, I don't know where I am wrong. I want a copy of your configuration.**
**Thank you 😃**

**Comment by mrossinek** [comment]
Hi! You can find my config here: https://gitlab.com/mrossinek/dotfiles/-/blob/master/nvim/.config/nvim/after/plugin/neorg.lua

The keybinds are not up-to-date at the time of writing but they are being overhauled right now anyways (see #308).

The `nvim-treesitter` config is here: https://gitlab.com/mrossinek/dotfiles/-/blob/master/nvim/.config/nvim/after/plugin/nvim-treesitter.lua

**Reply by ISouthRain** [reply]
Thank you !!
I noticed that the problem is in the Windows system !!
Configuration is Android Termux work !
But Windows system don't work !

**Reply by mrossinek** [reply]
Ah I see. I do not have a Windows machine. But I suggest you join our Discord! There we have quite a few people that use Windows and I sure you can find the help you need there :+1:
